.buildkite/bk.integration.pipeline.yml

```diff
@@ -383,6 +461,9 @@ steps:
           ASDF_KIND_VERSION: "0.27.0"
           DOCKER_VARIANTS: "{{matrix.variants}}"
           TARGET_ARCH: "amd64"
+          CUSTOM_IMAGE_TAG: "git-${BUILDKITE_COMMIT:0:12}"
+          CI_ELASTIC_AGENT_DOCKER_IMAGE: "docker.elastic.co/beats-ci/elastic-agent-cloud"
+          TF_VAR_integration_server_docker_image: "docker.elastic.co/beats-ci/elastic-agent-cloud:fips-${BUILDKITE_COMMIT:0:12}"
```

Comment:
Why is this specific to FIPS where everything else just seems to change the logic to allow pushing a locally built image; generic and unrelated to fips.

Reply:
good catch